Source/EasyNetQ.Tests/Mocking/MockBuilder.cs

```diff
@@ -72,6 +73,14 @@ public MockBuilder(string connectionString, Action<IServiceRegister> registerSer
                 return channel;
             });
 
+            if (testOutputHelper != null)
```

Comment:
From my point of view it should be done in following way: we can create one more implementation of IEasyNetQLogger and it will delegate all work to ITestOutputHelper.

But, in the liblog branch we are moving from self-written logging facade to LibLog and it will be better to not have such logging now.

Could you remove it?

Reply:
It's helpful to have the logger output from the tests, especially when testing error scenarios. I believe ITestOutputHelper has to come from the fixture, since XUnit is injecting it on instantiation, but you are right, from there it could be wrapped in something else before given to MockBuilder. I'll take a look at the liblog branch and remove the logging for now.